pkg/webui/console/components/collaborator-form/index.js

```diff
@@ -169,6 +188,7 @@ export default class CollaboratorForm extends Component {
           required
           autoFocus={!update}
           disabled={update}
+          warning={update && isYou ? sharedMessages.collaboratorWarningSelf : undefined}
```

Comment:
Please display this as a `<Notification warning content={sharedMessages.collaboratorWarningSelf} />` at the top of the form.

Reply:
<img width="710" alt="Screen Shot 2021-06-02 at 16 00 41" src="https://user-images.githubusercontent.com/181308/120494007-b9e49c80-c3bb-11eb-970d-794cf494bc46.png">